### PR TITLE
Finish db conn tests

### PR DESCRIPTION
**Pull request by squirrelo**
This should add the last tests needed. Should be green now!

### CODE REVIEW

**Comment by squirrelo** [comment]
We are green!

**Comment by wasade** [comment]
That's awesome!

@eldeveloper and @mortonjt, able to review?

**Comment by mortonjt** [comment]
:+1:

File: amgut/lib/data_access/ag_data_access.py [hunk 1]
```diff
@@ -435,6 +435,28 @@ def updateVioscreenStatus(self, survey_id, status):
...
+
+        Parameters
+        ----------
+        survey_id : str
+            The survey to get status for
+
+        Returns
+        -------
+        int
+            Vioscreen status
```
Comment (ElDeveloper) [inline amgut/lib/data_access/ag_data_access.py]: Cand you add a "Raises" section?

**Comment by ElDeveloper** [comment]
:+1:, just one comment!

**Comment by squirrelo** [comment]
Added, ready to go.

**Pull request merged**